### DISCUSSION TITLE
Import ODF data correct file structure in MTEX 5.8.0

### DISCUSSION THREAD

**Discussion by vitesh13** [mtex-toolbox/mtex#1326] [Ask Anything] [answered]
I have a file with list of euler angles in radians. Earlier I used MTEX 5.4.0 to plot ODF from such a file and it was able to import this data without any problems. 
Now I upgraded the MTEX to 5.8.0 and the import wizard with Import ODF data complains "could not detect file format".
So I guess something has changed with requirement for the file format. I have attached my file with orientations.
Can you suggest me about what is missing in it? Does it need separate weight etc to be mentioned?
[Orientations.txt](https://github.com/mtex-toolbox/mtex/files/7950096/Orientations.txt)

**Comment by kilir** [comment, marked as answer]
Hi,
assuming you want to load the list of orientations, not actually an ODF:
```
ori = orientation.load('Orientations.txt',crystalSymmetry(somthing),'ColumnNames',{'phi1','Phi','phi2'},'radians')
```
should work.

Cheers,
Rüdiger

**Reply by ddyyasfh** [reply]
Hello,Why do I prompt that the function or variable 'something' is not defined?

**Reply by kilir** [reply]
Hi,
you need to define it according to your data.
Rüdiger

**Comment by vitesh13** [comment]
Thanks a lot it worked!

**Comment by lfgmorales** [comment]
Hi

Would be helpful if you can explain your problem in a little more detailed. Also, there is a new myex version since last week, did you try that?

Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 11 Mar 2023, at 12:49, Crazetopher Dvaid Blazing IV ***@***.***> wrote:

﻿

Hello,Why do I prompt that the function or variable 'something' is not defined?

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1326#discussioncomment-5275362>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5INZY57IINOM7F3642D3W3RRENANCNFSM5M5VKHXA>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1326#discussioncomment-5275362", "url": "https://github.com/mtex-toolbox/mtex/discussions/1326#discussioncomment-5275362", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]
